anago

```diff
@@ -742,6 +743,23 @@ get_build_candidate () {
                                 $PARENT_BRANCH \
    || return 1
 
+  # When creating alphas, make sure that X.X.0 has been created first or warn.
```

Comment:
@david-mcmahon what about my original comment 

> Should we instead make sure "X.Y.0.rc.0" exists?

Sorry if this is answered indirectly and i didn't follow the logic :)